### ISSUE TITLE
Hm

### ISSUE THREAD

**Issue by kelio-mv** [Gabriela212022/aplicativo-1-js-#1]
A

**Comment by Gabriela212022** [comment]
Desculpa, eu faço isso por vários motivos. Não sei quem é você pra eu te
contar o motivo.

Em sex., 24 de mar. de 2023 17:25, Kélio ***@***.***>
escreveu:

**Comment by kelio-mv** [comment]
sou o Vibess, que te ajudou com o outline do input, estávamos conversando normalmente, e n vi motivo plausível para um block

**Comment by Gabriela212022** [comment]
Você simplesmente disse que tava com preguiça de testar o aplicativo, eu
prefiro nem me demorar em lugares onde sou tratada assim.

Em sex., 24 de mar. de 2023 18:38, Kélio ***@***.***>
escreveu:

**Comment by kelio-mv** [comment]
Não, eu disse que estava doente e que por isso eu li o código pelo celular
e não pelo computador, pois não queria levantar da cama assim.
Vc nem me pediu pra testar app nenhum.

Em sáb, 25 de mar de 2023 07:20, Gabriela ***@***.***>
escreveu:

**Comment by Gabriela212022** [comment]
Eu não li que tu tava doente e sim com preguiça, por isso eu achei melhor
poupar o meu tempo e o teu. Mas se for isso, melhoras.

Em sáb., 25 de mar. de 2023 09:04, Kélio ***@***.***>
escreveu:

**Comment by kelio-mv** [comment]
Por favor, delete essa issue, pois seu repositorio não sai da barra lateral do meu github, e eu não consigo deletar, só vc
